fvm/serviceaccount.go

```diff
@@ -20,5 +24,6 @@ transaction {
 // at the specified service address.
 func SystemChunkTransaction(serviceAddress flow.Address) *flow.TransactionBody {
 	return flow.NewTransactionBody().
-		SetScript([]byte(fmt.Sprintf(systemChunkTransactionTemplate, serviceAddress)))
+		SetScript([]byte(fmt.Sprintf(systemChunkTransactionTemplate, serviceAddress))).
+		AddAuthorizer(serviceAddress)
```

Comment:
Do we also need a way to disable signature checking ?

Reply:
there is separate context for system chunk without singature checking